src/lib/y2partitioner/actions/controllers/fstabs.rb

```diff
@@ -247,6 +258,16 @@ def known_fs_type?(entry)
           !entry.fs_type.is?(:auto) && !entry.fs_type.is?(:unknown)
         end
 
+        # Whether a fstab entry has a supported filesystem type
+        #
+        # Only some filesystem types are supported by YaST, see {Filesystems.all}.
```

Comment:
This comment, for exampled helps to the confusion that is being discussed in other comments. This is not a list of file-system supported by YaST, but a lit of file-systems offered by the Partitioner forms (which is a subset of the former).

Reply:
Right! I wanted to say Partitoner instead of YaST.